src/Stratis.Bitcoin.Features.BlockStore/ControllersAndClients/BlockStoreController.cs

```diff
@@ -13,8 +14,17 @@
 using Stratis.Bitcoin.Utilities.JsonErrors;
 using Stratis.Bitcoin.Utilities.ModelStateErrors;
 
-namespace Stratis.Bitcoin.Features.BlockStore.Controllers
+namespace Stratis.Bitcoin.Features.BlockStore.ControllersAndClients
 {
+    public static class BlockStoreRouteEndPoint
+    {
+        public const string GetBlock = "block";
+        public const string GetBlockCount = "GetBlockCount";
```

Comment:
Looks a bit out of place having the capitalisation here where the others do not

Reply:
Yes, agreed. Decided to not change it in this PR since it's more like a refactoring and changing API endpoint might case a ripple effect and break other projects that use this endpoint.